internal/server/data/group.go

```diff
@@ -114,11 +115,11 @@ func DeleteGroup(tx WriteTxn, id uid.ID) error {
 	return handleError(err)
 }
 
-func AddUsersToGroup(tx WriteTxn, groupID uid.ID, idsToAdd []uid.ID) error {
-	query := querybuilder.New("INSERT INTO identities_groups(group_id, identity_id)")
+func AddUsersToGroup(tx WriteTxn, groupID uid.ID, providerGroupName string, providerID uid.ID, idsToAdd []uid.ID) error {
+	query := querybuilder.New("INSERT INTO identities_groups(group_id, identity_id, provider_id, provider_group_name)")
```

Comment:
do we need something to ignore duplicates?

Reply:
There's an `ON CONFLICT DO NOTHING` in the query builder here that should handle this